src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -281,19 +281,21 @@ def supported_invocation_parameters(cls) -> list[InvocationParameter]:
                 max_value=2.0,
             ),
             IntInvocationParameter(
-                invocation_name="max_tokens",
+                invocation_name="max_completion_tokens",
                 canonical_name=CanonicalParameterName.MAX_COMPLETION_TOKENS,
-                label="Max Tokens",
+                label="Max Completion Tokens",
```

Comment:
We probably want to keep this, and just add max_completion_tokens.

Otherwise, I am not sure if we will correctly parse max tokens off of spans that already used that parameter name